mlshvch/hometask-4/my_hash.rb

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'digest'
+# class MyHash with custom implementation of Hash Table
+class MyHash
+  include Enumerable
+  EMPTY_HASH_TABLE_SIZE = 10
+
+  def initialize
+    create_empty_hash
+  end
+
+  def create_empty_hash(hash_table_size = EMPTY_HASH_TABLE_SIZE)
```

Comment:
Do you really need `hash_table_size` param? I think even after the whole clear of hash, hash should shrink the amount of buckets back to its default value.